hardhat.config.js

```diff
@@ -101,30 +106,23 @@ module.exports = {
   networks: {
     hardhat: {
       allowUnlimitedContractSize: !withOptimizations,
+      initialBaseFeePerGas: argv.coverage ? 0 : undefined,
     },
   },
   exposed: {
     imports: true,
     initializers: true,
     exclude: ['vendor/**/*'],
   },
-  docgen: require('./docs/config'),
-};
-
-if (argv.gas) {
-  require('hardhat-gas-reporter');
-  module.exports.gasReporter = {
+  gasReporter: {
+    enabled: argv.gas,
```

Comment:
this is supposed to be true by default, but removing it causes the gas reporter to not run.